src/components/qas/questions/qaCalculator/sizeCalculator/FindDiameter.jsx

```diff
@@ -32,7 +34,13 @@ export default function FindDiameter(props) {
         </span>
       </span>
       <span>= </span>
-      <span className={color}>{diameter ? `${diameter} km` : '?'}</span>
+      <span className={color}>
+        {diameter || '?'}
+        <QACalculatorIconUnit
+          className={color}
+          unit={<Unit type="diameter" />}
+        />
+      </span>
```

Comment:
This piece is now included in [EPO-3291] as to not worry about the merge conflict and because this was a simple boolean and text update.